azure/template.json

```diff
@@ -247,6 +247,10 @@
                                 "name": "DFE_SIGN_IN_SECRET",
                                 "value": "[parameters('dfeSignInSecret')]"
                             },
+                            {
+                                "name": "SETTINGS__DFE__SIGN__IN__SECRET",
+                                "value": "[parameters('dfeSignInSecret')]"
+                            },
                             {
                                 "name": "DFE_SIGN_IN_PROFILE",
```

Comment:
Can i reject based on you added "SETTINGS__DFE__SIGN__IN__SECRET"

and not update this file?

Reply:
feeling the need to reject you as deployment to via azure template will be broken